bundles/framework/announcements/service/util.js

```diff
@@ -0,0 +1,27 @@
+const RANGE_SEPARATOR = '\u2013';
+const TIME_OPTIONS = {
+    hour: '2-digit',
+    minute: '2-digit'
+};
+
+export const formatDate = (isoDateTime) => {
```

Comment:
Should we add this for Oskari.util.formatDate() instead of https://github.com/oskariorg/oskari-frontend/blob/1441ad3158b3780de05c171971a9d329ae0aa080/src/util.js#L775-L799